doc/offline-signing-tutorial.md

```diff
@@ -0,0 +1,259 @@
+# Offline Signing Tutorial
+
+Welcome to this tutorial on how to sign transactions offline using PSBT (Partially Signed Bitcoin Transactions). This tutorial will guide you through the process of securely signing Bitcoin transactions using an offline wallet and an online watch-only wallet.
+
+## Overview
+In this tutorial, we have two hosts: [offline] which is totally offline and without a copy of the blockchain and [online] which is a regular online node, both running Bitcoin 25.0
+
+In this example, we are going to generate a key on the offline wallet, and then send it 0.0009 BTC with an online(sending) wallet, then using the online wallet draft a transaction to send 0.0008 BTC to another online(receiving) wallet, then sign that with the offline wallet, and finally announce the signed transaction into the network using the online wallet.
+
+> [!NOTE]
+> Tested using Signet with a connected host machine and an offline docker container (representing the offline wallet) both running Signet, should also work with Regtest and Mainnet
+
+
+## Step 1: Sending money to the offline wallet
+
+### Create and Load Funds into the Online Sending Wallet
```

Comment:
I think the guide can be written without this online sending wallet. That would make it more practical too. I think starting this way is perhaps good for teaching a user about bitcoin but it I don't think it's necessary in a Bitcoin Core doc. The goal here in my opinion should be as close to real world usage as possible. So my suggestion would be: create the offline wallet, copy the descriptors to the online wallet, and then fund the wallet directly from the faucet.